package.json

```diff
@@ -48,7 +48,10 @@
         "storybook:build": "yarn storybook:stories:build && build-storybook",
         "docs:analyze": "wca analyze 'packages/*/src/index.ts' --format json --outFile documentation/custom-elements.json",
         "docs:build": "yarn build && yarn docs:ci",
+        "docs:build:staging": "yarn build && yarn docs:analyze && gulp docsDeploy && yarn storybook:build",
```

Comment:
So `docs:ci` only has to do the `gulp docsCompile` because it's compiling directly on the prod server? (or because there's a downstream CI task that does the prod deploy?)

Reply:
Now that I'm thinking about this more, is `gulp docsDeploy` the right name for that task? This isn't actually deploying to a server right? Since you're doing the storybook build after that seems unlikely.

Reply:
What a ball of yarn this one!

Updated to:
- `docs:build` and `docsBuildStaging` (the only documented command) this is what a user would use in a standard workflow as it prepares the docs site to run at the root of a server (localhost or Netlify, etc) and allows for direct inspection of a built documentation site
- `docs:ci` and `docsBuildProduction` which prepare the public documentation site, but a user would never need in standard use

The updated `gulp` naming should also better outline what those commands are used for...

😅